cmd/ebrelayer/relayer/cosmos.go

```diff
@@ -416,48 +286,17 @@ func (sub CosmosSub) handleBurnLockMsg(
 		"cosmosMsg", cosmosMsg,
 	)
 
-	client, auth, target, err := tryInitRelayConfig(sub, claimType)
+	valAddr, err := GetValAddressFromKeyring(txFactory.Keybase(), sub.ValidatorName)
 	if err != nil {
-		sub.SugaredLogger.Errorw("failed in init relay config.",
-			errorMessageKey, err.Error())
-		return
-	}
-
-	// Initialize CosmosBridge instance
-	cosmosBridgeInstance, err := cosmosbridge.NewCosmosBridge(target, client)
-	if err != nil {
-		sub.SugaredLogger.Errorw("failed to get cosmosBridge instance.",
-			errorMessageKey, err.Error())
-		return
+		sub.SugaredLogger.Infow(
+			"get the prophecy claim.",
+			"cosmosMsg", err,
+		)
 	}
 
-	maxRetries := 5
-	i := 0
+	signProphecy := ebrelayertypes.NewMsgSignProphecy(valAddr.String(), cosmosMsg.NetworkDescriptor,
+		cosmosMsg.ProphecyID, "", "")
 
-	for i < maxRetries {
-		err = txs.RelayProphecyClaimToEthereum(
-			cosmosMsg,
-			sub.SugaredLogger,
-			client,
-			auth,
-			cosmosBridgeInstance,
-		)
+	txs.SignProphecyToCosmos(txFactory, signProphecy, sub.CliContext, sub.SugaredLogger)
```

Comment:
The new solution is to send transaction to cosmos, we do it before because the old solution need send transaction to Ethereum, we often have some traffic issue. and we didn't have retry for Sifchain transaction. I think it is safe to remove retry since cosmos network much more efficient.